src/dash_layout.py

```diff
@@ -0,0 +1,164 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+
+
+def instructions():
+    """Instructions for using app"""
+    return """
+    SAEMI is a tool for obtaining a Size Analysis of particles
+    in Electron Microscopy Images. To obtain a size analysis,
+    first upload an Electron Microscopy image using the Upload
+    button below. The tool will then predict how to segment the
+    particles from the background using a model trained through
+    deep learning. After segmentation, the tool will label each
+    separate segment and count each segment's number of pixels.
+    This size distribution is then displayed in a histogram which
+    can be used to determine properties such as the mean size,
+    median size or standard deviation of sizes in the image.
+    Thank you for considering using this tool and any additional
+    feedback is always welcome. Good luck!
+    """
```

Comment:
Consider moving these instructions to a different file in a docs folder. This function could load a text file containing these instructions, which you can more easily edit over time.